qemu/tests/cfg/boot_nic_with_iommu.cfg

```diff
@@ -1,9 +1,15 @@
 - boot_nic_with_iommu:
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
     x86_64, i386:
         only q35
         no WinXP WinVista Win7 Win8 Win8.1 Win2003
         no Win2008 Win2008..r2 Win2012 Win2012..r2
         machine_type_extra_params = "kernel-irqchip=split"
+        virtio_dev_iommu_platform = on
+        virtio_dev_ats = on
+        iommu_caching_mode = on
```

Comment:
To use this, the precondition is `intel_iommu=yes`, suggest to move into `HostCpuVendor.intel`